hashable.cabal

```diff
@@ -84,10 +91,17 @@ library
     , bytestring  >=0.10.8.2 && <0.13
     , containers  >=0.5.10.2 && <0.7
     , deepseq     >=1.4.3.0  && <1.6
-    , filepath    >=1.4.1.2  && <1.5
```

Comment:
I'd put `filepath >= 1.4.1.2 && < 1.6` here and then `filepath < 1.5` into the `else`  (and delete `filepath` from the `if` part).
This style is slightly more concise.  And it is apparent that we _always_ depend on `filepath`.